zeebe/protocol/src/main/java/io/camunda/zeebe/protocol/record/value/ErrorType.java

```diff
@@ -22,6 +22,8 @@ public enum ErrorType {
 
   JOB_NO_RETRIES,
 
+  EL_NO_RETRIES,
```

Comment:
:wrench: We should use the full name instead of the acronym. 

For the JobKind, we also use the full name. 

```suggestion
  EXECUTION_LISTENER_NO_RETRIES,
```

Reply:
Done, thanks for suggesting!